Lib9c/Action/ItemEnhancement0.cs

```diff
@@ -114,7 +114,7 @@ public override IAccountStateDelta Execute(IActionContext context)
                 );
             }
 
-            var result = new ItemEnhancement.ResultModel
+            var result = new ItemEnhancement7.ResultModel
```

Comment:
It seems trivial, but I guess it may be needed to separating each ItemEnhancement class.